### ISSUE TITLE
[Feature Request] split up the "follow me" option

### ISSUE THREAD

**Issue by philcerf** [cecpk/RocketMAD#155]
Hey.

When follow me is disabled, then the location on the map updates rarely (and not very accurately (possibly using network location?))....

On the other hand... when follow me is enabled, it updates constantly but also forcibly centres the map all the time at that position.

Especially when  one does e.g. IV100% chasing by car/bike, this is pretty annoying... typical situation:
One is still further away from the destination but already wants to zoom in to see the details how to drive... then the centring is annoying.
But when one gets close, I want my exact position shown on the map, to see how far (e.g. in the road) I have to drive.

Ideal would be IMO the following:
- one option that allows to enable/disable use of GPS (i.e. "immediate" location update)
- one option that allows to get the automatic centring at the current position (whether GPS or network location)... even better would be if that centring could be configured to happen after n seconds.

Thanks.

**Comment by Axiangel** [comment]
You can always click the gmaps link to get a path to your destination instead and use that while travelling.   As you arrive, you open the map to check the exact location as well as verify the CP.

**Comment by philcerf** [comment]
That's not the same... the link shows me just one target...

Even when going for just one target (like a 100% IV Pokemon), the Google maps link might be far less useful. Often it stops at the nearest road Google recognises, while OSM may have far more detailed paths the go in a park.

So especially when the Pokemon is in a difficult to reach location, having the live (GPS) but not forcibly centered location on the RocketMAD, would be pretty helpful.


Same when one goes for mass hunting (like shiny hunting)... it's handy to see where exactly I am in contrast to the Pokemon (plural) of interest... while Google Maps would give me always just one.

**Comment by daangroot** [comment]
`When follow me is disabled, then the location on the map updates rarely` not true, if follow me is disabled the location only updates when you press the location button. Both will use the same location detection methods (GPS, network, or whatever). So follow me is not more accurate than the location button.

**Comment by philcerf** [comment]
Not does update for even without pressing the location button (IIRC)... but I mean that's just the point of this proposal:
I want the position to be regularly updated (without pressing any button or so).... just without centring the map. :-)

**Comment by na-ji** [comment]
Then it's a bug, not a feature request. Which browser, which device? If location works, I presume you already have https set up?

**Comment by daangroot** [comment]
No it's not a bug. He wants to automatically update the location marker position without the map panning to the location each time. I think it's a good feature.

**Comment by na-ji** [comment]
I thought it was already the case?

**Comment by daangroot** [comment]
No, right now, the location marker moves and the whole map moves to the users position.

**Comment by na-ji** [comment]
Ok now I understand!
